### PR TITLE
feat: add users endpoint to workflow

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #5 
Added method to return credentials yaml to `ArtipieYaml`, also corrected reading `credentials` section as we support it as  yaml mapping and as yaml sequence. Added `/users` endpoint to workflow.

### CODE REVIEW

File: src/main/java/com/artipie/front/settings/ArtipieYaml.java [hunk 1]
```diff
@@ -74,27 +80,48 @@ public BlockingStorage repoConfigsStorage() {
         );
     }
 
+    /**
+     * Credentials from config yaml mapping if the credentials type is file and
+     * this exists.
+     * @return Credentials file yaml
+     */
+    public Optional<YamlMapping> credentialsYaml() {
```
Comment (g4s8) [inline src/main/java/com/artipie/front/settings/ArtipieYaml.java:88]: Maybe `fileCredentials` (as we're using `file` config here)?

**Review by g4s8 — CHANGES_REQUESTED** [review]
@olenagerasimova please check my comments

**Comment by olenagerasimova** [comment]
@g4s8 thanks, corrected

**Review by g4s8 — APPROVED** [review]
@olenagerasimova thanks

**Pull request merged**